ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -667,7 +667,7 @@ export default defineComponent({
         (isTransferToSurvivingJointTenant.value ? localState.isValidDeathCertificate : true) &&
         (isTransferToExecutorProbateWill.value ? TransWill.isValidTransfer.value : true)
       )
-    }, { deep: true })
+    }, { immediate: true, deep: true })
```

Comment:
Will do an initial validation
( To support Draft data being resumed if the user makes no changes after resuming )